doc/content/en/docs/next/IDL Language/_index.md

```diff
@@ -374,3 +374,60 @@ protocol Simple {
 }
 ```
 Additional examples may be found in the Avro source tree under the `src/test/idl/input` directory.
+
+## IDE support
+
+There are several editores and IDEs that support Avro IDL files, usually via plugins.
```

Comment:
```suggestion
There are several editors and IDEs that support Avro IDL files, usually via plugins.
```

Reply:
Hrm... you're right.  In my memory there was a second capital, but the website of the editor says I'm wrong...